features/FEATURE_BLE/targets/TARGET_CORDIO/source/CordioPalGap.cpp

```diff
@@ -911,12 +984,11 @@ ble_error_t Gap::extended_scan_enable(
     if (enable) {
         uint32_t duration_ms = duration * 10;
 
-        DmScanModeExt();
         DmScanStart(
             scanning_phys.value(),
             DM_DISC_MODE_NONE,
             extended_scan_type,
-            filter_duplicates.value(), // TODO: cordio API incomplete ???
+            filter_duplicates.value(),
```

Comment:
Any more TODOs to update? 😜

Reply:
I'm not sure why I removed it, the comment still stands 😆 . These TODO's and FIXME are important in our implementations and internal files when we work with APIs that do not implement all Bluetooth requirements or when we deliver subsets of BT features.